cellprofiler/modules/identifyprimaryobjects.py

```diff
@@ -12,13 +11,21 @@
 import scipy.ndimage
 import scipy.sparse
 import skimage.morphology
-
 import cellprofiler.gui.help
 import cellprofiler.object
 import cellprofiler.setting
-import _help
 import threshold
 
+import numpy as np
+
+import cellprofiler.image as cpi
+import cellprofiler.module as cpm
+import cellprofiler.measurement as cpmeas
+import cellprofiler.preferences as cpprefs
+import cellprofiler.setting as cps
+from cellprofiler.setting import YES, NO
```

Comment:
These import aliases have been removed, and the full imports are declared above. You can delete lines 19-26.